ipv8/dht/payload.py

```diff
@@ -0,0 +1,240 @@
+from struct import pack, unpack, unpack_from, calcsize
```

Comment:
@egbertbouman , could you please consider using XDR (xdrlib) for packing/serialization? Contrary to Python structs, it is portable.

Reply:
For now, I'll stick with structs but perhaps in the future, this would be a nice addition.